tethys_gizmos/gizmo_options/map_view.py

```diff
@@ -255,6 +255,9 @@ class MVDraw(SecondaryGizmoOptions):
         controls(list, required): List of drawing controls to add to the map. Valid options are 'Modify', 'Delete', 'Move', 'Point', 'LineString', 'Polygon' and 'Box'.
         initial(str, required): Drawing control to be enabled initially. Must be included in the controls list.
         output_format(str): Format to output to the hidden text area. Either 'WKT' (for Well Known Text format) or 'GeoJSON'. Defaults to 'GeoJSON'
+        lineColor(str): User control for customizing the stroke color of annotation objects
```

Comment:
Be sure to update the doc strings to be consistent with the changes you made to the variable and argument names.